contracts/KYCPayloadDeserializer.sol

```diff
@@ -8,17 +8,17 @@
 import "./BytesDeserializer.sol";
 
 /**
- * A mix-in contract to decode different AML payloads.
+ * A mix-in contract to decode different signed KYC payloads.
  *
- * @notice This should be a library, but for the complexity and toolchain fragility risks involving of linking library inside library, we put this as a mix-in.
+ * @notice This should be a library, but for the complexity and toolchain fragility risks involving of linking library inside library, we currently use this as a helper method mix-in.
  */
 contract KYCPayloadDeserializer {
 
   using BytesDeserializer for bytes;
 
+  // @notice this struct describes what kind of data we include in the payload, we do not use this directly
```

Comment:
"@notice" is not meant to be used like this (https://ethereum.gitbooks.io/frontier-guide/content/natspec.html)